src/bitbox02.js

```diff
@@ -16,13 +16,23 @@ export async function getDevicePath() {
     const attempts = 10;
     for (let i=0; i<attempts; i++){
         let response;
+        let errorMessage;
         try {
             response = await fetch("http://localhost:8178/api/v1/devices", {
                 method: 'GET',
                 headers: {},
-            });
+            })
+            if (response.ok) {
+                response = response;
```

Comment:
Isn't this a no-op?

Reply:
Yep, definitely forgot to remove that one before pushing and leaving for a meeting :D

Reply:
Fixed now, my bad :)